recipes/cereal/all/conanfile.py

```diff
@@ -34,9 +34,15 @@ def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
         cmake = CMake(self)
         cmake.definitions["JUST_INSTALL_CEREAL"] = True
+        cmake.definitions["CEREAL_INSTALL"] = True
         cmake.configure()
         cmake.install()
-        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+        if tools.Version(self.version) < "1.3.1":
+            tools.rmdir(os.path.join(self.package_folder, "share"))
+        else:
+            tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
```

Comment:
tools.rmdir never raises. So it's fine here, very accurate, but with previous commit as well (simpler to maintain maybe).

Reply:
So you mean:
```
        if tools.Version(self.version) < "1.3.1":
            tools.rmdir(os.path.join(self.package_folder, "share"))
        else:
            tools.rmdir(os.path.join(self.package_folder, "share"))
            tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
```
if i got it right?

The previous commit was basically both `rmdir` commands, one with share and one with lib/cmake.

Reply:
Oh wait, I think I misunderstood. You mean roll back to just doing both without branching. Alright, I'd rather add a comment though because for the next person looking at the CMakeLists of the library, that line won't make much sense.

Reply:
as you wish, usually I don't comment too much this kind of stuff, eventually in git commit, to keep the code clean.